src/storage/txn/scheduler.rs

```diff
@@ -772,6 +779,16 @@ impl<E: Engine, L: LockManager> Scheduler<E, L> {
                 .process_write(snapshot, context)
                 .map_err(StorageError::from)
         };
+
+        // TODO: write bytes can be a bit inaccurate due to error requests or in-memory pessimistic locks.
```

Comment:
if the result is a error, no need to add write bytes. And do we need to add read bytes?

Reply:
here are process_write, don't need to add front read bytes in my view

Reply:
Ah, that's a problem. In the scheduler itself, most of its work is reading the engine. We probably should record the read bytes comtributed by the scheduler

Reply:
Can you give a suggestion where to record

Reply:
fetch these part of data from `statistics`

Reply:
fixed